package.json

```diff
@@ -50,6 +50,7 @@
     "nunjucks": "^3.0.1",
     "postcodesio-client": "billinghamj/postcodesio-client-node",
     "require-environment-variables": "^1.1.2",
+    "tunnel-agent": "^0.6.0",
```

Comment:
Hm, decided to update the policy for the moment. Snyk uses `npm update` instead, so the manual yarn upgrade doesn't quite do the trick.